cnitool/README.md

```diff
@@ -0,0 +1,16 @@
+# Overview
```

Comment:
So far, we've been keeping all markdown documentation (except README and SPEC) in the `./Documentation` folder.